python/cuvs/CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ project(
 option(FIND_CUVS_CPP "Search for existing CUVS C++ installations before defaulting to local files"
        OFF
 )
+set(CUVS_RAFT_CLONE_ON_PIN OFF)
```

Comment:
Why does this need to be set off?

Reply:
I copied the logic here from cuml https://github.com/rapidsai/cuml/blob/99b20f67c6dea4aba80da5787f64557dba30a1f8/python/cuml/CMakeLists.txt#L41 - but now I'm wondering if this might have been accidentally applied in cuml in https://github.com/rapidsai/cuml/pull/4990 , and not meant to be included there permanently.

Removed in latest commit

Reply:
Thanks for tracking that down, definitely looks to be accidental.